xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp

```diff
@@ -2635,12 +2635,14 @@ bool COutput::GLInit()
   }
 #endif
 
+#ifdef HAS_GL
```

Comment:
This is about all I can find, http://stackoverflow.com/questions/6883160/opengl-es-gl-texture-rectangle